docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:13.0-alpine
+    restart: always
+    ports:
+      - '5432:5432'
+    volumes: 
+      - db:/var/lib/postgresql/data
+      - ./db-init/init.sql:/docker-entrypoint-initdb.d/init.sql
+    env_file:
+      - ./server/.env.dev
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+ 
+
+  server:
+    build: ./server
+    image: ${IMAGE_NAME_FOR_RUN}
+    command: >
+      sh -c "
+      python ./manage.py migrate && \
+      python ./manage.py loaddata cpho/fixtures/dimension_lookups.yaml && \
+      python ./manage.py loaddata cpho/fixtures/periods.yaml && \
+      python ./manage.py runscript cpho.scripts.dev && \
+      exec gunicorn --bind 0.0.0.0:8080 --workers 1 --threads 8 --timeout 0 server.wsgi:application"
+    ports:
+      - 8080:8080
```

Comment:
Where these `command` steps necessary for running the tests? Or was this for testing the image locally? Should be able to remove it all, including the exposed port, and still run tests in this image. Maybe even give an empty `command` value to overwrite the image default, or replace it with the command to execute the tests

Reply:
I guess I'm assuming that this `docker-compose.yaml` is intended to be single purpose for running tests in cloud build. If it is, drop these commands and maybe rename it something like `docker-compose.cloud-build-test-env.yaml`.

Reply:
[Changed network flag to external: true to be able to work in cloud bu…](https://github.com/PHACDataHub/cpho-phase2/commit/f671166efc11b7d2548eac3ec27e6e1b418bb455) 

Was unsure if we should simplify the run line exec gunicorn --bind 0.0.0.0:8080 --workers 1 --threads 8 --timeout 0 server.wsgi:application  to something like exec gunicorn  server.wsgi:application as other flags not needed here, or to keep it as close to run command as possible?

Reply:
I might be missing something, but I meant that if all this container does is run tests then `command` could just be `coverage run -m pytest` and ports could be closed. The Django test runner handles any migrations if necessary and `server/tests/conftest.py` loads the necessary fixture data. `cpho.scripts.dev` and starting gunicorn isn't necessary for running the tests either.

Reply:
You're absolutely right - iteratively getting there!